src/test/java/org/jenkinsci/main/modules/cli/auth/ssh/UserPropertyImplTest.java

```diff
@@ -29,7 +29,6 @@ private void testRoundtrip(String publicKey) throws Exception {
         foo.addProperty(new UserPropertyImpl(publicKey));
         r.configRoundtrip(foo);
         assertEquals(publicKey, foo.getProperty(UserPropertyImpl.class).authorizedKeys);
-        // integration testing w/ sshd-module is in hudson.cli.CLITest.strictHostKey in core
```

Comment:
Obsolete after #38 now that https://github.com/jenkinsci/sshd-plugin/blob/da71afe504618988831c66063c82687ae481bee4/src/test/java/org/jenkinsci/main/modules/cli/auth/ssh/CLITest.java#L124 is here.

Reply:
(even after #44)